frontend/src/util/index.ts

```diff
@@ -37,6 +37,27 @@ export function characterClass(characters: string): string {
     return `[${characters}]`;
 }
 
+/** Checks whether or not a given string is contained in another string */
+export const localeContains = (mainString: string, stringToCheck: string) => {
+    if (stringToCheck === "") {
+        return true;
+    }
+    if (!stringToCheck || !mainString.length) {
+        return false;
+    }
```

Comment:
Uhm. `!stringToCheck` is only true if it is empty right? But you check that above already. Further, `!mainString.length` is... equal to just `!mainString`? :thinking: